core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -177,6 +185,11 @@ export class OpportunitiesListComponent {
     const lastPageNumber = Math.ceil(opportunities.length / pageSize);
     return activePageNumber >= lastPageNumber && !moreResults;
   }
+
+  onChangeLanguage(languageCode: string): void {
+    this.languageCode = languageCode;
```

Comment:
Is the language code actually used anywhere in this component?

Reply:
I searched and looks like it isn't used in the component nor in the html.

The `this.languageCode` in `contributions-and-review.component.ts` is being used, but is unrelated to this one.